docs/Governance/REP/index.md

```diff
@@ -13,5 +23,4 @@ Community members that receive a REP boost need to submit a REP boost proposal o
 
 If you believe that you should receive REP for a DXdao contribution, contact [Chris](https://daotalk.org/u/powers/summary) on DAOtalk.org to be nominated. DXdao welcomes nominating new community members for REP, and we are looking for new effective ways to allocate REP, while still honoring DXdao’s merit-based governance system. If you have a suggestion on how to allocate REP, please state it in the #governance channel on the [dx_dao](https://keybase.io/team/dx_dao) Keybase or [DAOtalk.org](https://daotalk.org/c/dx-dao/15).
 
-Full-time workers may earn up to 4% REP for paid contributions to DXdao. A worker may request 0.1667% of REP for each month she has worked.
-
+Full-time workers may earn up to 4% REP for paid contributions to DXdao. A worker may request 0.1667% of REP for each month she has worked.
```

Comment:
Also it seems kinda weird for here to say she..

Reply:
These sections had been written by Tammy, and a small group of people reviewed and agreed that there was no issue with the language. "Its the girls turn!"

Reply:
I think we should put "he/she" since we shouldn't emphasise gender in any way...

Reply:
I will discuss, and make the change if it makes sense

Reply:
From Tammy, who wrote most of the Contributor Hub section: 

"So, I used she because in academic circles and law she is used instead of he. He/she is very awkward to constantly repeat, and many academics don't want to go with "he" lol"

Reply:
Additional: 

"Thanks for asking though. Do what the majority prefers, of course, but the only other option is he. He/she sounds so awkward."

Reply:
okay i'll merge it so it's not a blocker.... but sounds super weird still to say just she in my opinion when i was reading it it looked like typo...but on the other hand not fair to say just he....